src/Surfnet/StepupMiddleware/CommandHandlingBundle/Tests/SensitiveData/SensitiveDataTest.php

```diff
@@ -61,17 +61,6 @@ public function sensitiveDataToSerialise()
                     ->forget(),
                 ['CommonName' => CommonName::unknown(), 'Email' => Email::unknown()],
             ],
-            'DocumentNumber' => [
```

Comment:
Shouldn't this be tested with the `VettingType` object / `withVettingType` method?
And are you sure the sensitive data is going to work with the old event stream? Please test this with the old `document_number` set as sensitive data.

Reply:
Did you validate this?

Reply:
Yup, tested that, and all seems to work as intended.